README.md

```diff
@@ -34,3 +34,31 @@ To execute all tests, run the following command:
 ```sh
 make test
 ```
+
+## Adding a new loader
```

Comment:
I suggest the following:

1. Write the loader: explain the loader signature and add a simple example
2. Add the loader: explain how to add it to index.js with an example